src/main/java/oss/fosslight/domain/CoMailManager.java

```diff
@@ -232,13 +232,12 @@ public boolean sendMail(CoMail bean) {
     			if(_reMakeContents.isEmpty()) {
     				return false;
     			}
-    			
-    			// vulnerability_prj_recalc_oss_info 이름으로 매핑 정보를 설정한다.
+
```

Comment:
@soimkim  It was for code readability. That comment seemed to repeat what the code did, which was not needed to be saved as a comment. 

Making comments should be restricted to a certain extent, I think,  because the more comments are created, the more developers need time to read code. Is it okay to delete all these comments? Otherwise, feel free to let me know. I will roll it back!

Reply:
Ok. let’s delete this comment.